### ISSUE TITLE
Documentation updates

### ISSUE THREAD

**Issue by drewatk** [brandonmp/gatsby-source-google-sheets#22]
We need to update documentation to make this a bit better on beginners

- How to query for data, and what fields are available: #21 #13 
- Limitations of the google sheets api: #20 #17

**Comment by drewatk** [comment]
Also add new "type guessing" logic from #19

**Comment by krerkkiat** [comment]
@drewatk Does anyone taking this issue yet? May I be any help?

**Comment by Waschnick** [comment]
I still haven't figured out how to query data with GraphQL. I did setup everything with credentials and it builds, but how do I write the API queries?

**Comment by krerkkiat** [comment]
@Waschnick If you still looking into how to query, please have a look at changes in #29 (or a rendered README.md [here](https://github.com/krerkkiat/gatsby-source-google-sheets/tree/revise-documentation)).

**Comment by stewest** [comment]
YES! That helps a lot, thanks! https://github.com/krerkkiat/gatsby-source-google-sheets/tree/revise-documentation

**Comment by doublejosh** [comment]
Can someone please document what the graphQL query is supposed to look like to use this plugin?

**Comment by stewest** [comment]
Hi @doublejosh 

Did you see https://github.com/brandonmp/gatsby-source-google-sheets/pull/29/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R50

It will depend on the shape of your data sheet, i.e. the column names.

**Comment by doublejosh** [comment]
@stewest The confusing part is the table/type name, not the columns. In the example it's `allGoogleSheetPublicationsRow`

I was only able to discover mine by hitting the local GraphQL endpoint `http://localhost:8000/___graphql`

It seems to generate a name from the title of the sheet and the tab name, and makes choices about capitalization.

In my case... `allGoogleSpreadsheetWwWebsiteContentVideos`

```
spreadsheetName: "WWWebsiteContent",
typePrefix: "GoogleSpreadsheet",
```

**Comment by stewest** [comment]
Hi @doublejosh yes, indeed, odd name. I think it needs to make it unique, as one could have multiple sources. Getting the name from GraphQL is the way. Good luck with your project!
